docs/apib/node_bmc.apib

```diff
@@ -0,0 +1,15 @@
+FORMAT: 1A
+
+# XClarity Nodes
+
+# PUT /nodes/6B538D351B8211E193F65CF3FC6E4030/bmc
```

Comment:
For the enabled test, I am only checking to see if the PUT request was actually made.  I can add a test that validates the response in some way. I will work on that.